account.go

```diff
@@ -154,6 +160,12 @@ type TransferSchedule struct {
 	MonthAnchor uint64   `json:"monthly_anchor"`
 }
 
+type TOSAcceptanceParams struct {
```

Comment:
can you please add a comment for this? All public structs and methods should have comments